programmers/서울에서-김서방찾기.js

```diff
@@ -0,0 +1,10 @@
+function solution(seoul) {
+    let answer;
```

Comment:
문자열로 사용할꺼면 미리 지정해주시는게 좋습니다.
```suggestion
    let answer = "";
```